### PR TITLE
Upgrade to Ubuntu 20.04, Elixir 1.11-otp-23

### PR DESCRIPTION
**Pull request by chadfennell**
And now I see @lelder1 has a PR already up for his merge. Sigh.

Note: the image size went from 1GB uncompressed on my local machine to 1.6GB. Looks like esl-erlang is the main culprit.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,9 @@
-FROM ubuntu:16.04
-ENV ELIXIR_VERSION=1.10.3-otp-22
+FROM ubuntu:20.04
+ENV ELIXIR_VERSION=1.11.3-otp-23
+
+# Set Timezone so TZ Info doesn't hault the build
+ENV DEBIAN_FRONTEND="noninteractive"
+ENV TZ="America/Chicago"
```
Comment (ConnorRigby) [inline Dockerfile]: :japanese_goblin: any reason not to use UTC?

**Review by ConnorRigby — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (chadfennell) [reply Dockerfile]: Not really - picked a zone close to us. UTC might be a better option.

Comment (chadfennell) [reply Dockerfile]: That goblin is amazing.

**Comment by adkron** [comment]
Always UTC unless something else is explicitly required. 

Amos King
CEO
Binary Noggin

File: Dockerfile (hunk 1 above)
Comment (chadfennell) [reply Dockerfile]: Looks like it builds UTC by default, so just rolling with that.

**Pull request merged**